src/components/TransactionFlow/TxReceiptTotals.tsx

```diff
@@ -97,40 +99,21 @@ export const TxReceiptTotals = ({
       <div className="TransactionReceipt-row">
         <div className="TransactionReceipt-row-column">
           <SIcon type="tx-sent" alt="Sent" />
-          {translate('TOTAL')}
+          {`${translateRaw('TOTAL')}:`}
         </div>
         <div className="TransactionReceipt-row-column rightAligned">
-          {asset.type === 'base' ? (
```

Comment:
Do we not want to show token transfers in this way anymore?

Reply:
Correct. We're moving TxStatus to look more like an etherscan tx status panel for now.